ilc/client/isActiveFactory.js

```diff
@@ -0,0 +1,52 @@
+export default function isActiveFactory({
+    singleSpa,
```

Comment:
I think that you can use imports to handle:
- `singleSpa`
- `handlePageTransaction`
- `slotWillBe`